pythonsd/templates/pythonsd/organizers.html

```diff
@@ -5,39 +5,40 @@
 
 
 {% block main %}
-<div class="container mt-3">
+<div class="container mt-4">
 <h1>Organizers</h1>
 
 <p>If you would like to reach out, please contact the <a href="mailto:sandiegopython-organizers@googlegroups.com">Python SD Organizers</a>.</p>
 
 
 {% if organizers %}
-<div class="row row-cols-2 row-cols-md-4">
+<div class="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-12 mb-16 ">
   {% for organizer in organizers %}
-  <div class="col mb-4">
-    <div class="card">
-      <img src="{{ organizer.photo.url }}" class="card-img-top" alt="{{ organizer.name }}">
-      <div class="card-body">
-        <h5 class="card-title">{{ organizer.name }}</h5>
-        <ul class="list-inline">
+    <div class="flex flex-col rounded border border-slate-200 max-w-64 md:max-w-80 mx-auto">
+      <img src="{{ organizer.photo.url }}" class="flex-none" alt="{{ organizer.name }}">
```

Comment:
```suggestion
      <img src="{{ organizer.photo.url }}" class="flex-none aspect-square object-cover" alt="{{ organizer.name }}">
```
This preserves the aspect ratio if the uploaded file is not 1:1.

Reply:
Definitely a defensive improvement. Thanks.